lib/pages/network_troubleshoot/port_scan_page.dart

```diff
@@ -77,22 +78,22 @@ class _PortScanPageState extends State<PortScanPage>
       _openPorts.clear();
     });
     if (_type == ScanType.single) {
-      PortScanner.isOpen(
+      PortScannerFlutter.isOpen(
         _targetIPEditingController.text,
         int.parse(_singlePortEditingController.text),
       ).then((value) {
         _handleEvent(value);
         _handleOnDone();
       });
     } else if (_type == ScanType.top) {
-      _streamSubscription = PortScanner.customDiscover(
+      _streamSubscription = PortScannerFlutter.customDiscover(
```

Comment:
Why isn't it called `PortScanner` there is no need to specify that it is from a different library.

Reply:
Although it can be kept same, but then users have to add alias to import statement when using both. To simplify it I'm suffixing `Flutter` word

Reply:
They aren't supposed to import both of the packages, flutter version should extended and replace dart package

And expose same functionality and more outside for the developer